frontend/src/utils/index.js

```diff
@@ -317,6 +320,57 @@ export function getProjectDetails (project) {
   }
 }
 
+export function getProjectQuotaStatus (projectQuota) {
```

Comment:
I would move these two functions into the store. In the tests we can then import the raw getters.